### ISSUE TITLE
Showing new listings for Thursday, 6 February 2025

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#83]
## Elimination of substrate-induced FMR linewidth broadening in the epitaxial system YIG-GGG by microstructuring
- **Keywords:** ['hybrid quantum']
 - **Authors:** David Schmoll, Rostyslav O. Serha, Jaganandha Panda, Andrey A. Voronov, Carsten Dubs, Michal Urbánek, Andrii V. Chumak
 - **Subjects:** Subjects:
Mesoscale and Nanoscale Physics (cond-mat.mes-hall)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2502.02978
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2502.02978
 - **Abstract**
 Modern quantum technologies and hybrid quantum systems offer the opportunity to utilize magnons on the level of single excitations. Long lifetimes, low decoherence rates, and a strong coupling rate to other subsystems propose the ferrimagnet yttrium iron garnet (YIG), grown on a gadolinium gallium garnet (GGG) substrate, as a suitable platform to host magnonic quantum states. However, the magnetic damping at cryogenic temperatures significantly increases due to the paramagnetic character and the highly inhomogeneous stray field of GGG, as recent experiments and simulations pointed out. Here, we report on temperature dependent ferromagnetic resonance (FMR) spectroscopy studies in YIG-GGG thin-films with different sample geometries. We experimentally demonstrate how to eliminate the asymmetric stray field-induced linewidth broadening via microstructuring of the YIG film. Additionally, our experiments reveal evidence of a non-Gilbert like behavior of the linewidth at cryogenic temperatures, independent of the inhomogeneous GGG stray field.
## TensorQC: Towards Scalable Distributed Quantum Computing via Tensor Networks
- **Keywords:** ['hybrid quantum']
 - **Authors:** Wei Tang, Margaret Martonosi
 - **Subjects:** Subjects:
Emerging Technologies (cs.ET); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2502.03445
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2502.03445
 - **Abstract**
 A quantum processing unit (QPU) must contain a large number of high quality qubits to produce accurate results for problems at useful scales. In contrast, most scientific and industry classical computation workloads happen in parallel on distributed systems, which rely on copying data across multiple cores. Unfortunately, copying quantum data is theoretically prohibited due to the quantum non-cloning theory. Instead, quantum circuit cutting techniques cut a large quantum circuit into multiple smaller subcircuits, distribute the subcircuits on parallel QPUs and reconstruct the results with classical computing. Such techniques make distributed hybrid quantum computing (DHQC) a possibility but also introduce an exponential classical co-processing cost in the number of cuts and easily become intractable. This paper presents TensorQC, which leverages classical tensor networks to bring an exponential runtime advantage over state-of-the-art parallelization post-processing techniques. As a result, this paper demonstrates running benchmarks that are otherwise intractable for a standalone QPU and prior circuit cutting techniques. Specifically, this paper runs six realistic benchmarks using QPUs available nowadays and a single GPU, and reduces the QPU size and quality requirements by more than $10\times$ over purely quantum platforms.
## Hybrid Quantum Neural Networks with Amplitude Encoding: Advancing Recovery Rate Predictions
- **Keywords:** ['hybrid quantum']
 - **Authors:** Ying Chen, Paul Griffin, Paolo Recchia, Lei Zhou, Hongrui Zhang
 - **Subjects:** Subjects:
Computational Finance (q-fin.CP); Machine Learning (cs.LG); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2501.15828
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2501.15828
 - **Abstract**
 Recovery rate prediction plays a pivotal role in bond investment strategies, enhancing risk assessment, optimizing portfolio allocation, improving pricing accuracy, and supporting effective credit risk management. However, forecasting faces challenges like high-dimensional features, small sample sizes, and overfitting. We propose a hybrid Quantum Machine Learning model incorporating Parameterized Quantum Circuits (PQC) within a neural network framework. PQCs inherently preserve unitarity, avoiding computationally costly orthogonality constraints, while amplitude encoding enables exponential data compression, reducing qubit requirements logarithmically. Applied to a global dataset of 1,725 observations (1996-2023), our method achieved superior accuracy (RMSE 0.228) compared to classical neural networks (0.246) and quantum models with angle encoding (0.242), with efficient computation times. This work highlights the potential of hybrid quantum-classical architectures in advancing recovery rate forecasting.
